proposals/improved-interpolated-strings.md

```diff
@@ -361,9 +361,13 @@ lowering for `i` is performed as follows:
 of a larger expression `e`, any components of `e` that occurred before `i` will be evaluated as well, again in lexical order.
 2. `Fc` is called with the length of the interpolated string literal components, the number of _interpolation_ holes, any previously evaluated arguments, and a `bool` out argument
 (if `Fc` was resolved with one as the last parameter). The result is stored into a temporary value `ib`.
+    1. The length of the literal components is calculated after replacing any _open_brace_escape_sequence_ with a single `{`, and any _close_brace_escape_sequence_
```

Comment:
Is `open_brace_escape_sequence` something that's defined elsewhere? This is the first mention in this document.
If yes, consider adding link to reference.

Reply:
Yes, it's defined in the lexical specification of interpolated strings. I use a number of components from the existing lexical and expression specification of interpolated strings throughout this document, I don't think it's necessary to link back for every single term from those components.a